lib/providers/quoters/RoutingApiQuoter.ts

```diff
@@ -1,15 +1,22 @@
 import { TradeType } from '@uniswap/sdk-core';
 import { NATIVE_NAMES_BY_ID } from '@uniswap/smart-order-router';
+import { AxiosError } from 'axios';
+import axiosRetry from 'axios-retry';
 import querystring from 'querystring';
-import axios from './helpers';
 
-import { AxiosError } from 'axios';
 import { NATIVE_ADDRESS, RoutingType } from '../../constants';
 import { ClassicQuote, ClassicRequest, Quote } from '../../entities';
 import { log } from '../../util/log';
 import { metrics } from '../../util/metrics';
+import axios from './helpers';
 import { Quoter, QuoterType } from './index';
 
+// Retry quote requests to the RoutingAPI up to 3 times if they 5xx
+axiosRetry(axios, {
+  retries: 3,
```

Comment:
think in slack discussion we should do 1 retry?

Reply:
removed this